packages/mobile/src/app/reducers.ts

```diff
@@ -119,6 +121,11 @@ export const appReducer = (
         ...state,
         locked: false,
       }
+    case Actions.SET_SESSION_ID:
+      return {
+        ...state,
+        sessionId: action.sessionId,
```

Comment:
should this be reset to '' in the rehydrate reducer above?

Reply:
Yeah good call